Resources/Textures/Clothing/Uniforms/color.rsi/meta.json

```diff
@@ -2,8 +2,10 @@
   "version": 1,
   "size": {
     "x": 32,
-    "y": 32
-  },
+    "y": 32,
+      },
+  "license": "CC-BY-SA-3.0/CCA-NC-SA-3.0",
```

Comment:
Please don't put NC icons in the same RSI files as non-NC icons... That'll probably cause problems in the future.

Reply:
it means we can't tell which are which...

also `CCA-` isn't a valid license segment, guess typo'd?